cmd/kubeadm/test/token_test.go

```diff
@@ -65,3 +65,35 @@ func TestCmdTokenGenerateTypoError(t *testing.T) {
 		t.Error("'kubeadm ex token genorate' (a deliberate typo) exited without an error when we expected non-zero exit status")
 	}
 }
+
+// kubeadmReset executes "kubeadm reset" and restarts kubelet.
+func kubeadmReset() error {
+	_, _, err := RunCmd(kubeadmPath, "reset")
+	return err
+}
+
+func TestCmdInitToken(t *testing.T) {
+	var initTest = []struct {
+		args     string
+		expected bool
```

Comment:
I can't tell from the name if this is an expected error or an expected success, please make it more clear